src/app/content/highlights/components/ShowMyHighlights.tsx

```diff
@@ -57,7 +57,10 @@ class ShowMyHighlights extends Component<{}, { showGoToTop: boolean }> {
         ref={this.myHighlightsBodyRef}
         data-testid='show-myhighlights-body'
       >
-        <Filters />
+        <Styled.HighlightsToolbar>
+          <Filters />
+          <Styled.HiglightsPrintButton />
```

Comment:
i think you might want to add the `HiglightsPrintButton` within the filters component, because filters has a two row design and the print button is supposed to be on the first row. currently the way you have this the button is taking away space where the second row of filters is supposed to go